### PR TITLE
Added PIP Install Commands for Python Modules

### PR DESCRIPTION
**Pull request by tso914**
Added PIP Install commands for the Python Modules we use.

### CODE REVIEW

**Comment by tso914** [comment]
New PIP Install Commands for Python modules

File: Dockerfile [hunk 1]
```diff
@@ -1,48 +1,60 @@
...
-
-# Create a 'data' volume for mounting external post data
-VOLUME /data
-
-# Create a 'source' volume for mounting external python source files.
-VOLUME /src
-
-# bash as a default command (but allow to be overridden by anything else supplied on CMD line)
-CMD bash
+FROM centos:latest
```
Comment (stephenconnolly1) [inline Dockerfile]: @tso914  - not sure what's gone on here, but the diffs are showing up as Deletes and Adds. Can you check that you haven't changed the line-ending settings on the dockerfile.

File: Dockerfile [hunk 2]
```diff
@@ -1,48 +1,60 @@
...
+# Install libpostal
+RUN git clone https://github.com/openvenues/libpostal && \
+    cd libpostal && \
+    ./bootstrap.sh && \
+    ./configure --prefix=/usr --datadir=/opt/libpostaldata && \
+    make && \
+    make install
+
+# Install pypostal
+RUN pip3.6 install postal
```
Comment (stephenconnolly1) [inline Dockerfile]: Each RUN command creates a new image layer, which can be expensive and slow.
Might be more efficient to have 


    RUN pip3.6 install postal \
        happybase \
        pandas \ 
      ...

**Comment by tso914** [comment]
Hi Stephen, 

I have made the changes to the end of lines and also corrected the PIP Install command to be in one run. I acidently uploaded a .txt version of the file which I don't seem to be able to delete. Is it a permission thing? can you delete it?

Thanks,

**Review by stephenconnolly1 — APPROVED** [review]
Changes made as required

**Comment by stephenconnolly1** [comment]
@tso914 - Changes approved and merged. The build should appear in Docker Hub in about an hour or so. I would suggest you pull that down on your laptop, save as a tarball, import into the env and then within Dev, do a Docker load and then docker push up to Artifactory. 
Give me a shout if you need help - the docker commands can be a bit fiddly, especially for pushing up to artifactory with the right label.

**Pull request merged**